docs/JestObjectAPI.md

```diff
@@ -314,6 +314,46 @@ test('moduleName 2', () => {
 });
 ```
 
+Using `jest.doMock()` with ES6 modules requires additional steps:
```

Comment:
No it doesn't. Using `jest.doMock` with ES6 `import`s does. Please clarify this, because people may get wrong idea that they need to install some babel plugin to actually test a module.

This should clearly state something like: "use if you don't want to `require` your ES6 modules"

Reply:
Good point, changed.